basics/keeping-up-to-date/backup.md

```diff
@@ -41,6 +41,7 @@ tar -czf <file_name>.tar <folder_to_save>
 ```
 
 For instance:
+
```

Comment:
Hello, can you explain why adding these white spaces is a good thing? I am not sure it provides value 🤔

Reply:
This is simply to separate each block/paragraph visually. I can undo this if I need to.